charts/fhir-server/templates/config-internal.yaml

```diff
@@ -0,0 +1,209 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fhir-server-config-internal
+data:
+  datasources.xml: >
+    {{- if eq .Values.db.type "db2" }}
+    TODO
+    {{- else }}
+    <server>
+      <!-- ============================================================== -->
+      <!-- TENANT: default; DSID: default; TYPE: read-write               -->
+      <!-- ============================================================== -->
+      <dataSource id="fhirDefaultDefault" jndiName="jdbc/fhir_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true" validationTimeout="30s">
```

Comment:
I know. we. added `isolationLevel="TRANSACTION_READ_COMMITTED"` for. db2, do we need this for postgres?

Reply:
I'm pretty sure that Robin said its not needed for postgresql...something about the default being different for db2 and derby.